packages/framework/theming-ramp/src/Typography.types.ts

```diff
@@ -85,8 +83,7 @@ export type FontWeightValue = '100' | '200' | '300' | '400' | '500' | '600' | '7
  */
 export interface IFontWeights {
   light: FontWeightValue;
-  semiLight: FontWeightValue;
-  medium: FontWeightValue;
+  regular: FontWeightValue;
   semiBold: FontWeightValue;
   bold: FontWeightValue;
 }
```

Comment:
I'm wondering if all these font weights still make sense in the new type ramp

Reply:
Good point. Perhaps I should just keep 'regular' and 'semibold'? Also, should it be 'semibold' or 'semiBold'? I feel that 'semibold' is easier to type.

Reply:
Updated fontWeights to 'regular' and 'semiBold' in recent commits.